### ISSUE TITLE
web admin: display switches take long time

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#5562]
**Describe the bug**
When you have an important number of switches (tested with 226 switches and 44 switch groups), display switches take long time in web admin.

**To Reproduce**
1. Import an important number of switches
2. Try to display first 25 switches 

=> You have to wait several seconds

**Screenshots**
I joined a screenshot with time elapsed between each query and answer:
![Capture d’écran de 2020-06-10 09-04-03](https://user-images.githubusercontent.com/10830260/84237600-edb71a00-aaf9-11ea-977a-2e5d0ea6c886.png)

**Expected behavior**
Display first 25 switches should be quick.

**Additional context**
I noticed that size of answer `OPTIONS` is quite big because it's contain all switches modules definition.

**Comment by nqb** [comment]
@satkunas, I can provide you my switches.conf (comes from a customer).

**Comment by nqb** [comment]
For information, third API call (`/api/v1/config/switch_groups?sort=id&fields=id,description`) return 25 results like fourth API call even if `limit=25` is not used.

**Comment by nqb** [comment]
With logs in debug, I see many many lines like this in `packetfence.log`:
```
Jun 10 11:50:32 pfdeb9dev packetfence: pfperl-api(1380) DEBUG: cache get for namespace='configfiles', key='/usr/local/pf/conf/switches.conf', cache='Redis:l1_cache', time='0ms': HIT (CHI::Driver::_log_get_result
```

**Comment by nqb** [comment]
@jrouzierinverse or @satkunas, any idea what can cause this behavior ?

**Comment by jrouzierinverse** [comment]
This was fixed by e60640274fdfb5ab3dfed01e53d56d15f2ba4852
